cfpb/capital-framework
# Update Docs for Capital Framework v7.0.0

Updates the docsite to use the latest release from Capital Framework.

## Changes

- Updates cf-expandables to 7.0.0
- Updates cf-tables to 5.0.0
- Updates cf-atomic-component to 2.0.0

## Testing

- Run `npm start`
- Open http://localhost:3000/components/cf-expandables and ensure the expandables work as before and the bug in the accordion example is fixed
- Open http://localhost:3000/components/cf-tables and ensure the tables work as before
